src/System.Xml.XDocument/src/System/Xml/Linq/OrderedNodeEnumerable.cs

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
```

Comment:
Going by this discussion: https://github.com/dotnet/coreclr/pull/1329#discussion_r36218185, you probably have right to put your initials after copyright and purge Microsoft, if you don't work for Microsoft.